### ISSUE TITLE
Add new/Edit -> Add new/Edit crash

### ISSUE THREAD

**Issue by YossarianXXII** [JonasJurczok/factorio-todo-list#17]
Reproduce error:
Create new  task.
Hit edit/add new
Hit edit/add new again 

What is needed is to **make sure the 'todo_add_frame' is open only once per player**
I may try to fix it myself, but don't know the factorio api.

```
Error while running event Todo-List::on_gui_click (ID 1)
Gui element with name todo_add_frame already present in the parent element.
stack traceback:
	__Todo-List__/UI.lua:86: in function 'create_add_edit_frame'
	__Todo-List__/todo.lua:84: in function 'edit_task'
	__Todo-List__/todo.lua:186: in function 'on_gui_click'
	__Todo-List__/control.lua:16: in function <__Todo-List__/control.lua:15>
```

```
Error while running event Todo-List::on_gui_click (ID 1)
Gui element with name todo_add_frame already present in the parent element.
stack traceback:
	__Todo-List__/UI.lua:86: in function 'create_add_edit_frame'
	__Todo-List__/todo.lua:175: in function 'on_gui_click'
	__Todo-List__/control.lua:16: in function <__Todo-List__/control.lua:15>
```

Something in the means like this
```
function todo.create_add_edit_frame(player)
    local gui = player.gui.center
	**if 'gui contains element with name 'todo_add_frame' return**
    local frame = gui.add({
        type = "frame",
        name = "todo_add_frame",
        caption = {"todo.add_title"},
        direction = "vertical"
    })
```

**Comment by JonasJurczok** [comment]
Hi, sorry for not responding for such a long time.. I had a child and was.. distracted. But hopefully I will have more time for this project again starting from now..

This bug is pretty high on my list and should be easily fixable. 

Sorry again for ignoring this ticket for so long :/

**Comment by Tarrke** [comment]
I have a fix for this one. Not sure if I should make another PR as I have one open. This (branch)[https://github.com/Tarrke/factorio-todo-list/tree/bug/add-new-edit] contains the fix.

**Comment by JonasJurczok** [comment]
Yes please just create a new PR :)
